client/battle/CBattleInterface.cpp

```diff
@@ -105,12 +105,17 @@ CBattleInterface::CBattleInterface(const CCreatureSet *army1, const CCreatureSet
 {
 	OBJ_CONSTRUCTION;
 
-	if (!curInt)
+	if(spec)

```

Comment:
Oh. This is certainly something useless. Probably remaining of my duel mode fixing attempts.